packages/web-react/docs/DocsFormFieldGroup.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { ReactNode } from 'react';
+
+interface DocsFormFieldGroupProps {
+  children: ReactNode;
+}
+
+const DocsFormFieldGroup = ({ children }: DocsFormFieldGroupProps) => (
```

Comment:
There is a class `docs-FormFieldGrid` so I think it would be better to use the name of the class for the component, e.g. `DocsFormFieldGrid`, or rename the class (but this will be needed in other packages as well).